### ISSUE TITLE
Cognitoidentityprovider can't recognize new parameters during update

### ISSUE THREAD

**Issue by Benedikt1992** [crossplane-contrib/provider-aws#1305]
### What happened?
When creating a resource in the `cognitoidentityprovider` group it is not possible to add parameters after the resource has been created. They are not recognized as a change in the spec.

### How can we reproduce it?

Create a `UserPool` with minimal values. Add a new value (e.g. AccountRecoverySetting). It won't update the resource.

Create:
```yaml
apiVersion: cognitoidentityprovider.aws.crossplane.io/v1alpha1
kind: UserPool
metadata:
  name: example
spec:
  forProvider:
    region: us-east-1
    poolName: examplePool
  providerConfigRef:
    name: example
```
Update with:
```yaml
apiVersion: cognitoidentityprovider.aws.crossplane.io/v1alpha1
kind: UserPool
metadata:
  name: example
spec:
  forProvider:
    region: us-east-1
    poolName: examplePool
    accountRecoverySetting:
      recoveryMechanisms:
        - name: verified_email
          priority: 1
  providerConfigRef:
    name: example
```

### What environment did it happen in?
All versions

**Comment by wotolom** [comment]
works as of https://github.com/crossplane-contrib/provider-aws/pull/1533

**Comment by MisterMX** [comment]
Closing, since this seems to be fixed in #1533.
